ariadne/cardano/src/Ariadne/Wallet/Cardano/WalletLayer.hs

```diff
@@ -1,12 +1,32 @@
 module Ariadne.Wallet.Cardano.WalletLayer
     ( -- * Kernel
       passiveWalletLayerComponent
-    , passiveWalletLayerWithDBComponent
+    , passiveWalletLayerInMemoryDBComponent
     -- * We re-export the types since we want all the dependencies
     -- in this module, other modules shouldn't be touched.
     , module Types
+    -- * Tests use the bracket interface throughout
+    , bracketKernelPassiveWallet
     ) where
 
+import Control.Monad.Component (runComponentM)
+import Control.Monad.IO.Unlift (MonadUnliftIO, withRunInIO)
+import System.Wlog (Severity)
+
+import Ariadne.Wallet.Cardano.Kernel (PassiveWallet)
+import Ariadne.Wallet.Cardano.Kernel.Keystore (Keystore)
 import Ariadne.Wallet.Cardano.WalletLayer.Kernel
-  (passiveWalletLayerComponent, passiveWalletLayerWithDBComponent)
+  (passiveWalletLayerComponent, passiveWalletLayerInMemoryDBComponent)
+import Ariadne.Wallet.Cardano.WalletLayer.Types (PassiveWalletLayer)
 import Ariadne.Wallet.Cardano.WalletLayer.Types as Types
+
+bracketKernelPassiveWallet
```

Comment:
Similar comment as above. I think this function should be moved to tests or its name should reflect that it uses in-memory DB.